ndmg/stats/qa_fast.py

```diff
@@ -1,92 +1,65 @@
+    
+    parameters
+    ---------------
+    csf: str
+    the path of csf nifity image
+    gm: str
+    the path of gm nifity image
+    wm: str
+    the path of wm nifity image
+    outdir: str
+    the path to save QA graph
+    
     """
-    pad_width = (((max_dim-image.shape[0])//2,(max_dim-image.shape[0])//2),((max_dim-image.shape[1])//2,(max_dim-image.shape[1])//2))
-    padded_image = np.pad(image, pad_width=pad_width, mode='constant', constant_values=pad_val)
-
-    return padded_image
-
-def plot_overlays(gm, csf, wm, cmaps=None, minthr=2, maxthr=95, edge=False):
-    '''
-    gm: 3d nparray of gm
-    csf: 3d nparray of csf
-    wm: 3d nparray of wm
-    '''
-    plt.rcParams.update({"axes.labelsize": "x-large", "axes.titlesize": "x-large"})
-    foverlay = plt.figure()
-    plt.title(f'Qa for fast (brain volume:{gm.shape}) \n\n', fontsize=20)
+    
+    # load data
+    gm_data = nb.load(gm).get_data()
+    csf_data = nb.load(csf).get_data()
+    wm_data = nb.load(wm).get_data()
+    
+    # set Color map
+    cmap1 = LinearSegmentedColormap.from_list('mycmap1', ['white', 'blue'])
+    cmap2 = LinearSegmentedColormap.from_list('mycmap2', ['white', 'magenta'])
+    cmap3 = LinearSegmentedColormap.from_list('mycmap2', ['white', 'green'])
+    
+    
+    overlay = plt.figure()
+    overlay.set_size_inches(12.5, 10.5, forward=True)
+    plt.title(f'Qa for FAST(segments a 3D image of the brain into different tissue types)\n (scan volume:{gm_data.shape}) \n', fontsize=22)
     plt.xticks([])
     plt.yticks([])
     plt.axis('off')
-
-    if gm.shape != csf.shape:
-        raise ValueError("Brains are not the same shape.")
-    if cmaps is None:
-        cmap1 = LinearSegmentedColormap.from_list("mycmap1", ["white", "magenta"])
-        cmap2 = LinearSegmentedColormap.from_list("mycmap2", ["white", "green"])
-        cmap3 = LinearSegmentedColormap.from_list("mycmap3", ["white", "blue"])
-        cmaps = [cmap1, cmap2, cmap3]
-
-    if csf.shape == (182, 218, 182):
-        x = [78, 90, 100]
-        y = [82, 107, 142]
-        z = [88, 103, 107]
-    else:
-        shap = csf.shape
-        x = [int(shap[0] * 0.35), int(shap[0] * 0.51), int(shap[0] * 0.65)]
-        y = [int(shap[1] * 0.35), int(shap[1] * 0.51), int(shap[1] * 0.65)]
-        z = [int(shap[2] * 0.35), int(shap[2] * 0.51), int(shap[2] * 0.65)]
+    
+    # Determine whether the input data types are consistent. If they are inconsistent, an error is reported.
+    if gm_data.shape != csf_data.shape:
+        raise ValueError("GM and CSF are not the same shape.")
+    elif gm_data.shape != wm_data.shape:
+        raise ValueError("GM and WM are not the same shape.")
+    elif wm_data.shape != csf_data.shape:
+        raise ValueError("WM and CSF are not the same shape.")
+    
+    # Set the 3D matrix cutting position in three directions
+    shape = csf_data.shape
+    slice = [0.35, 0.51, 0.65]
```

Comment:
`slice` is actually a [python3 default function](https://docs.python.org/3/library/functions.html#slice) that stores slices to index with. Name this something else.